src/Json.Schema.ToDotNet/PropertyInfoDictionary.cs

```diff
@@ -630,7 +643,9 @@ private string GetUnqualifiedTypeName(string typeName, out string namespaceName)
                 namespaceName = null;
             }
 
-            return unqualifiedTypeName.ToPascalCase();
+            namespaceName = namespaceName == "System" ? null : namespaceName;
+
+            return typeName.StartsWith("System.") ? unqualifiedTypeName : unqualifiedTypeName.ToPascalCase();
```

Comment:
>unqualifiedTypeName [](http://example.com/codeflow?start=52&length=19)

This is because we don't want to use `Int`, should be `int`.
All system types are already with correct case we don't need to manually do ToPascalCase.